index.html

```diff
@@ -62,41 +62,41 @@ <h1 id="title">Welcome to Our Survey Form</h1>
         placeholder="number"
         min="0"
         max="100"
-      />
+      /> 
       <br /><br />
       <label for="dropdown">Work Experience:</label><br />
       <select id="dropdown">
         <option value="first">0-5</option>
-        <option value="second">5-10</option>
-        <option value="third">10-15</option>
+        <option value="second">5-10 </option>
+        <option value="third">10-15 </option>
         <option value="fourth">More Than 15</option>
       </select>
       <br /><br />
       <label for="Gender">Gender</label><br />
       <p>
-        <input type="radio" name="gender" value="male" checked /> Male<br />
+        <input type="radio" name="gender"  value="male" checked /> Male<br />
 
-        <input type="radio" name="gender" value="female" />female><br />
-      </p>
+        <input type="radio" name="gender"  value="female" />female><br />
+      </p> 
       <br /><br />
       <label for="Gender">Check Those You Choose</label><br />
-      <input type="checkbox" name="Favourite Foods" value="Sushi" />Sushi<br />
-      <input type="checkbox" name="Favourite Foods" value="Rice" />Rice<br />
+      <input type="checkbox" name="Favourite Foods"  value="Sushi" />Sushi<br />
+      <input type="checkbox" name="Favourite Foods"  value="Rice" />Rice<br /> 

```

Comment:
It's good to wrap elements like these ones in a list using `<ul>` and `<li>` tags.